analysis/shamanelemental/src/modules/talents/Stormkeeper.tsx

```diff
@@ -8,10 +8,10 @@ import { STATISTIC_ORDER } from 'parser/ui/StatisticBox';
 import React from 'react';
 
 const AFFECTED_ABILITIES = [
-  SPELLS.LIGHTNING_BOLT_OVERLOAD.id,
-  SPELLS.LIGHTNING_BOLT.id,
-  SPELLS.CHAIN_LIGHTNING_OVERLOAD.id,
-  SPELLS.CHAIN_LIGHTNING.id,
+  SPELLS.LIGHTNING_BOLT_OVERLOAD,
+  SPELLS.LIGHTNING_BOLT,
+  SPELLS.CHAIN_LIGHTNING_OVERLOAD,
+  SPELLS.CHAIN_LIGHTNING,
```

Comment:
Good catch, I've gotten got by this exact bug a few times 😭 - something weird about the `SPELLS` index lets it dodge the typechecker